tests/kafkatest/tests/client/pluggable_test.py

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.utils.util import wait_until
+
+from kafkatest.tests.verifiable_consumer_test import VerifiableConsumerTest
+
+class PluggableConsumerTest(VerifiableConsumerTest):
```

Comment:
So, it looks like this doesn't directly verify a pluggable version (e.g. the dummy one which is actually guaranteed to be there). Is this intentional? I'm guessing that means this is only useful when trying to sanity check your client implementation settings and isn't actually useful on, e.g., a nightly basis? I wonder if we should `@ignore` it (or add some similar annotation). It's probably pretty cheap to run so not a big deal either way, but I'm wondering if we want to have some approach for tests like these that are not helpful nightly but helpful when integrating other clients.